newIDE/app/src/MainFrame/EditorTabs/DropIndicator.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+import ThemeConsumer from '../../UI/Theme/ThemeConsumer';
+
+const styles = {
+  columnDropIndicator: {
+    borderRight: '1px solid #18dcf2',
+    borderLeft: '1px solid #18dcf2',
```

Comment:
As the color will be overriden, might be worth putting a non descriptive one (black) or just use two CSS properties to avoid specifiying it entirely.